src/extensions/functions/onboard/kickoffs/kickoffCLI.ts

```diff
@@ -1,7 +1,7 @@
-import { SolidarityRunContext } from '../../../../types'
+import { SolidarityRunContext, SolidarityRule } from '../../../../types'
 
-export default async (context: SolidarityRunContext): Promise<void> => {
-  const { print } = context
-  print.info(`kickoff CLI function`)
-  // return { rule: 'cli' }
+export default async (context: SolidarityRunContext): Promise<SolidarityRule> => {
+  // const { print } = context
+  // print.info(`kickoff CLI function`)
```

Comment:
is this no longer needed?

Reply:
No, I believe the print statement was just there for debugging while implementing this feature.